packages/core/src/sanitization/sanitization.ts

```diff
@@ -155,8 +155,11 @@ export function ɵɵsanitizeScript(unsafeScript: any): TrustedScript|string {
  *
  * @codeGenApi
  */
-export function ɵɵtrustConstantHtml(html: string): TrustedHTML|string {
-  return trustedHTMLFromString(html);
+export function ɵɵtrustConstantHtml(html: TemplateStringsArray): TrustedHTML|string {
+  if (html.length !== 1) {
+    throw new Error('unexpected interpolation in trusted HTML constant');
```

Comment:
May be we can add more context for users, so that it's easier to find the problem?
```suggestion
    throw new Error(`Unexpected interpolation in trusted HTML constant: ${html.join('?')}`);
```

Reply:
Another comment: can we add some tests to trigger this code path (to avoid regressions in the future)?

Reply:
Thanks, done!